apps/portal/controllers/dashboards.jag

```diff
@@ -21,6 +21,11 @@ var fresh;
             return;
         }
 
+        if (userDomain !== (urlDomain || superDomain)) {
+            response.sendError(401, 'designer access not authorized of this tenant');
```

Comment:
The message should be started with uppercase, and the message is grammatically wrong. Please fix this.